app/src/main/java/ch/epfl/sdp/peakar/user/profile/NewProfileActivity.java

```diff
@@ -80,27 +122,30 @@ private void hideUI(boolean self, boolean friends) {
 
     /**
      * Fill list view.
-     * TODO get list from DB?
+     * TODO show correct points and correct date
```

Comment:
I think that the correct country should also be filled in this method, maybe add it in the TODOs.

Reply:
Good point